scripts/imgtool/keys/ecdsa.py

```diff
@@ -191,3 +209,84 @@ def sign(self, payload):
             return sig
         else:
             return sig
+
+
+class ECDSA384P1Public(ECDSAPublicKey):
+    """
+    Wrapper around an ECDSA (p384) public key.
+    """
+    def __init__(self, key):
+        super().__init__(key)
+        self.key = key
+
+    def shortname(self):
+        return "ecdsap384"
+
+    def sig_type(self):
+        return "ECDSA384_SHA384"
```

Comment:
Why not `ECDSA384_SHA256`?

Reply:
ECDSA384_SHA256 is actually fairly pointless, as you're only getting about 128-bits of "security" out of the hash function, using the larger curve is not giving you any additional security.